src/newsView.js

```diff
@@ -0,0 +1,81 @@
+const newsApi = require("./newsApi");
+const newsModel = require("./newsModel");
+
+class newsView {
+  constructor(model = new newsModel(), api = new newsApi()) {
+    this.model = model;
+    this.api = api;
+    this.mainContainerEL = document.querySelector("#container");
+    this.newsSearchEL = document.querySelector("#news-search");
+    this.newsSearchSubmitEL = document.querySelector("#news-search-submit");
+
+    this.newsSearchSubmitEL.addEventListener("click", () => {
+      this.api.loadNews(this.newsSearchEL.value, (data) => {
+        this.model.setNews(data);
+        console.log(this.model.getNews());
+        this.clearDuplicateNews;
+        this.displayNews();
+        this.newsSearchEL.value = "";
+      });
+    });
+  }
+
+  displayNews() {
+    this.clearDuplicateNews();
+    this.clearDuplicateImages();
```

Comment:
Could you combine these into one parent element and then remove the parent element?